# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: intl/uconv/tests/unit/test_utf8_illegals.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c intl/uconv/tests/unit/test_utf8_illegals.js
--- a/intl/uconv/tests/unit/test_utf8_illegals.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/intl/uconv/tests/unit/test_utf8_illegals.js	Tue Feb 12 16:08:25 2019 +0000
@@ -101,7 +101,7 @@
 	 "nsIConverterInputStream",
 	 "init");
   var channel = NetUtil.newChannel({uri: dataURI, loadUsingSystemPrincipal: true});
-  var testInputStream = channel.open2();
+  var testInputStream = channel.open();
   var testConverter = new ConverterInputStream(testInputStream,
 					       "UTF-8",
 					       16,